dist/bundle/webxr.js

```diff
@@ -6535,13 +6535,13 @@ var cbor = createCommonjsModule(function(module) {
 /**
  * This files defines the HoloPlayClient class and Message class.
  *
- * Copyright (c) [2019] [Looking Glass Factory]
+ * Copyright (c) [2024] [Looking Glass Factory]
  *
  * @link    https://lookingglassfactory.com/
  * @file    This files defines the HoloPlayClient class and Message class.
  * @author  Looking Glass Factory.
- * @version 0.0.8
- * @license SEE LICENSE IN LICENSE.md
+ * @version 0.0.11
+ * @license SEE LICENSE IN LICENSE.txt
```

Comment:
This file doesn't actually have an extension in the repo

Reply:
the bundle is generated with code from holoplaycore.js, which is a different repo.

Reply:
we now include dependencies.txt in the bundle folder which includes the licenses of third party packages we use in this repo, including the holoplay-core license.